builder/chroots.go

```diff
@@ -80,6 +82,416 @@ func readBuildChrootsConfig(path string) (*buildChrootsConfig, error) {
 	return cfg, nil
 }
 
+func updateBundlePackages(set bundleSet, packages *sync.Map) {
+	for _, b := range set {
+		for k := range b.DirectPackages {
+			deps, ok := packages.Load(k)
+			if !ok {
+				continue
+			}
+
+			for _, d := range deps.([]string) {
+				d = strings.Trim(d, " ")
+				if len(d) > 0 {
+					b.DirectPackages[d] = true
+				}
+			}
+		}
+	}
+}
+
+func isBannedPath(path string) bool {
+	if path == "/" {
+		return true
+	}
+
+	bannedPaths := []string{
```

Comment:
Since this doesn't change, should this get pulled up so it's not reinitialized every time `isBannedPath` is called (which, if I understand correctly, is for every file in the build)

Reply:
Absolutely, I'll do that. Good call.

Reply:
done.